addons/udes_stock/tests/test_stock_quant_package.py

```diff
@@ -122,59 +122,6 @@ def test06_get_reserved_quantity(self):
         )
         self.assertEqual(self.test_package.get_reserved_quantity(), 15)
 
-    def test07_create_simple_picking_from_package(self):
```

Comment:
These changes are for "create_picking from quants/packages" commit, not "stock_quant_package" commit.